infercnvpy/tl/_copykat.py

```diff
@@ -0,0 +1,106 @@
+import infercnvpy as cnv
+import numpy as np
+import pandas as pd
+import scipy
+from scipy.sparse import issparse
+from anndata import AnnData
+from scanpy import logging, AnnData
+
+def copyKAT(
```

Comment:
```suggestion
def copykat(
```

let's use lowercase everywhere to be consistent with python code-style.

Reply:
on it